docs/docs/guides/cli/how-to-compile-your-lightdash-project.mdx

```diff
@@ -0,0 +1,44 @@
+import NewWorkflowExistingActions from './../../snippets/assets/new-workflow-existing-actions.jpg';
+import GithubRun from './../../snippets/assets/github-run.png';
+import GithubSecrets from './../../snippets/github-secrets.mdx';
+
+# Test your changes with Lightdash compile
+
+If you've connected Lightdash to GitHub, you can setup a `github action` and get Lightdash to compile your project and test your changes whenever you open a pull request. 
+
+Adding this Lightdash compile action will compile your dbt project's .yml files and check to see if there are any errors that will break your Lightdash project. For example, a metric that references a dimension that doesn't exist.
+
+### Step 1: add the credentials to Github secrets
+
+<GithubSecrets />
+
+### Step 2: Create deploy.yml workflow in Github
+
+Go to your repo, click on `Actions` menu.
+
+If you don't have any GitHub actions, you'll just need to click on `Configure`
+
+![Github actions page](./assets/github-actions.png)
+
+If you have some GitHub actions in your repo already, click on `New workflow`, then select `setup a workflow yourself`.
+
+<img src={NewWorkflowExistingActions} width="952" height="203" style={{display: "block", margin: "0 auto 20px auto"}}/>
+
+Now copy [this file](https://github.com/lightdash/cli-actions/blob/main/compile.yml) from our [cli-actions](https://github.com/lightdash/cli-actions) repo.
+
+:::info
+If you only use a subset of your dbt models in Lightdash, then you'll want to specify that subset in your file [here](https://github.com/lightdash/cli-actions/blob/d91de777ed3668b537acd141b734eca75def4e52/compile.yml#L65). For example, to only compile models with the tag `lightdash`, you would change this line to: `run: lightdash compile --select tag:lightdash --project-dir "$PROJECT_DIR" --profiles-dir . --profile prod || lightdash compile --select tag:lightdash --project-dir "$PROJECT_DIR" --profiles-dir .`
```

Comment:
@ZeRego can you double check that this is correct please? 🙏

Reply:
looks good 👍